sections/03_categorize.md

```diff
@@ -139,66 +123,45 @@ setting, and model fine-tuning needed is very problem and dataset-specific.
 Moreover, this type of tasks often depends on both lesion localization and
 appearance that pose challenges for CNN-based approaches. Straightforward
 attempts to capture useful information from full-size images in all three
-dimensions simultaneously make applications of standard neural network
-architectures computationally unfeasible due to the curse of dimensionality.
-Instead, two dimensional models are often used to either process image slices
+dimensions simultaneously via standard neural network architectures were computationally unfeasible.
+Instead, two dimensional models were used to either process image slices
```

Comment:
The usage of dimensionality (two-dimensional, 2D, etc) is a bit hard for me to follow. See what you think when you go through.